nodeAPI/src/classes/deleteClass.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+import { convertStringToArray } from '../models/utilities';
+import { convertArrayToString } from '../models/utilities';
```

Comment:
[Nit]. You can just do { convertArrayToString, convertStringToArray } instead of two different imports